internal/generate/clusterserviceversion/clusterserviceversion.go

```diff
@@ -165,6 +169,14 @@ func (g *Generator) generate() (base *operatorsv1alpha1.ClusterServiceVersion, e
 	if g.FromVersion != "" {
 		base.Spec.Replaces = genutil.MakeCSVName(g.OperatorName, g.FromVersion)
 	}
+	base.Spec.RelatedImages = make([]operatorsv1alpha1.RelatedImage, 0, len(g.RelatedImages))
+	for name, image := range g.RelatedImages {
+		base.Spec.RelatedImages = append(base.Spec.RelatedImages, operatorsv1alpha1.RelatedImage{Name: name, Image: image})
+	}
+	// ensure deterministic order
+	sort.SliceStable(base.Spec.RelatedImages, func(i, j int) bool {
+		return strings.Compare(base.Spec.RelatedImages[i].Name, base.Spec.RelatedImages[j].Name) > 0
+	})
```

Comment:
I like this.

Reply:
The alternative is passing a slice of structs with a name/image instead of a map so we can maintain the same order that the user provides in the environment. I went with this since we didn't need to expose a kubes type (`operatorsv1.RelatedImage`) or define a new type. Curious as to what you think.